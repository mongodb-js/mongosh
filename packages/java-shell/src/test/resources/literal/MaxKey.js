// command checkResultClass
new MaxKey();
// command checkResultClass
MaxKey()