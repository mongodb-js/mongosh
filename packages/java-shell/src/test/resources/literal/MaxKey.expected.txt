MaxKeyResult: { "$maxKey" : 1}
MaxKeyResult: { "$maxKey" : 1}